platforms/android/library-compose/src/main/java/io/element/android/wysiwyg/compose/RichTextEditor.kt

```diff
@@ -146,21 +161,14 @@ private fun RealEditor(
         }
 
         view
-    }, update = { view ->
-        Timber.d("RealEditor's update block called, recomposing")
-        view.applyStyleInCompose(style)
-        view.styleConfig = style.toStyleConfig(view.context)
-        view.typeface = typeface
-        view.rustErrorCollector = RustErrorCollector(onError)
-        view.mentionDisplayHandler = object : MentionDisplayHandler {
-            override fun resolveMentionDisplay(text: String, url: String): TextDisplay {
-                return resolveMentionDisplay?.invoke(text, url) ?: TextDisplay.Plain
-            }
-
-            override fun resolveAtRoomMentionDisplay(): TextDisplay {
-                return resolveRoomMentionDisplay?.invoke() ?: TextDisplay.Plain
-            }
-
+    }, update = remember(style, typeface, mentionDisplayHandler) {
```

Comment:
This workaround seems to work, thank you! Please could you add some comments to explain the problem and why this is needed? I've created an issue to link to as well https://github.com/matrix-org/matrix-rich-text-editor/issues/856. Please feel free to add more detail that could help us in future.

Reply:
(friendly reminder to please link to the issue in the code comments)